src/test/java/com/googleinterns/zoomtube/servlets/AuthenticationServletTest.java

```diff
@@ -38,77 +45,92 @@ public void cleanUp() {
   }
 
   @Test
-  public void doGet_loggedIn_expectTrue() throws ServletException, IOException {
+  public void doGet_loggedIn_expectTrue() throws Exception {
     authService.setEnvIsLoggedIn(true);
     authService.setEnvAuthDomain("example.com");
     authService.setEnvEmail("test@example.com");
+    StringWriter content = new StringWriter();
+    PrintWriter writer = new PrintWriter(content);
+    when(response.getWriter()).thenReturn(writer);
 
     servlet.doGet(request, response);
 
-    assertThat(response.getContentType()).isEqualTo("application/json;");
-    String json = response.getContentAsString();
+    verify(response).setContentType("application/json;");
+    String json = content.toString();
     Gson gson = new GsonBuilder().registerTypeAdapterFactory(GenerateTypeAdapter.FACTORY).create();
     AuthenticationStatus status = gson.fromJson(json, AuthenticationStatus.class);
     assertThat(status.loggedIn()).isTrue();
   }
 
   @Test
-  public void doGet_loggedIn_expectFalse() throws ServletException, IOException {
+  public void doGet_loggedIn_expectFalse() throws Exception {
     authService.setEnvIsLoggedIn(false);
     authService.setUp();
+    StringWriter content = new StringWriter();
+    PrintWriter writer = new PrintWriter(content);
+    when(response.getWriter()).thenReturn(writer);
 
     servlet.doGet(request, response);
 
-    assertThat(response.getContentType()).isEqualTo("application/json;");
-    String json = response.getContentAsString();
+    verify(response).setContentType("application/json;");
+    String json = content.toString();
     Gson gson = new GsonBuilder().registerTypeAdapterFactory(GenerateTypeAdapter.FACTORY).create();
     AuthenticationStatus status = gson.fromJson(json, AuthenticationStatus.class);
     assertThat(status.loggedIn()).isFalse();
```

Comment:
nit: can you separate the assert?

Reply:
The `verify` and json stuff is part of the assertions/test section